expo/package.json

```diff
@@ -11,9 +11,6 @@
     "prepublishOnly": "expo-module prepublishOnly",
     "expo-module": "expo-module"
   },
-  "dependencies": {
```

Comment:
so we move it to `peerDependencies` so that user has to install it manually to be visible to Expo?

Reply:
the reason for the move is that putting it as a dependency of `expo-datadog` means that `@datadog/mobile-react-native` podspec's won't be detected by react-native autolinking.
For it to be detected it has to be a dependency of the app project.
Putting it as a peer dependency will trigger warning if it's not installed or at the wrong version.